pkg/cni/k8sclient/k8sclient.go

```diff
@@ -0,0 +1,36 @@
+/*
+ Copyright 2023, NVIDIA CORPORATION & AFFILIATES
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+     http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package k8sclient
+
+import (
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+// FromKubeconfig returns a kubernetes client created from provided kubeconfig path
+func FromKubeconfig(kubeconfigPath string) (*kubernetes.Clientset, error) {
```

Comment:
shouldn't we just copy past 
https://github.com/k8snetworkplumbingwg/multus-cni/blob/b9acfeb6b7fd37c6541327b5b3e1363ab9d57068/pkg/k8sclient/k8sclient.go#L398-L445 ?

Reply:
current code was borrowed from whereabouts after cleanup.
its pretty straight forward as we only need to support one option here : create client form kubeconfig path.

what logic do you think is needed to be additionally borrowed from the snipped you provided ?